<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContainerFrame</class>
 <widget class="QWidget" name="ContainerFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ContainerFrame</string>
  </property>
  <widget class="QGroupBox" name="groupBox_group">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Alignment</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_aligment">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_flat">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>flat</string>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>209</width>
      <height>139</height>
     </rect>
    </property>
    <widget class="QComboBox" name="combo_h">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_v">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_size">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>水平</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>垂直</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>SizeAdjustPolicy</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_display">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>181</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox显示</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
